go.mod

```diff
@@ -4,7 +4,7 @@ go 1.14
 
 require (
 	github.com/braintree/manners v0.0.0-20160418043613-82a8879fc5fd // indirect
-	github.com/cloudevents/sdk-go/v2 v2.0.1-0.20200608152019-2ab697c8fc0b
+	github.com/cloudevents/sdk-go/v2 v2.0.1-0.20200630063327-b91da81265fe
```

Comment:
this should gret bumped to `v2.1.0`

Reply:
I didn't touch this, I just ran `./hack/update-deps.sh --upgrade` after locking PKG